pytest_spec/plugin.py

```diff
@@ -15,6 +15,18 @@ def pytest_addoption(parser):
         help='Print test result in specification format'
     )
 
+    # register config options
+    parser.addini(
+        'spec_header_format',
+        default='{path}:{class_name}',
```

Comment:
Based on lib test default value should be set to
```
'{path}::{class_name}'
```
Documentation should be changed as well